Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04
+
+COPY . $HOME/ILLIXR/
```

Comment:
How does this actually work? Does the GitHub server always puts ILLIXR in $HOME/ ?

Reply:
The github server does a checkout of the PRs branch and then this line in the dockerfile copies that checked out code into the image.